currency/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Simple Currency System
+Help Site: [This is Optional]/<> <-This is Not Optional
+
+There Will be More Commands In the Future :)
+
+Simple Currency System with Changeable Name/Emoji Money.
+
+It Works With Some Of The Fun Commands that work with Currency, ex: slotsMachine.cc.lua or guessNumber.cc.lua
+
+These Commands Are Standalone, You can add Only one, all or two
+
+# Some Current Commands It Have:
+1. **-balance [User/Member]**
+   It Gets The Current Balance of A User Member or if no arguments given, The Users Balance, ex: !balance
+1. **-addbalance <User/Member> <Amount>** 
+   It adds Balance To A User/Member, ex: !addbalance @Gas_Infinity 100
+1. **-rob <User/Member>** 
+   It robs a Random Amount from a minAmount to the max Money The Player Have, It Detects if you or The Player Have The Min Amount and If you`re Robbing to Yourself, Also it Have A Very Simple         Cooldown System. ex: !rob @Gas_Infinity

```

Comment:
```suggestion
   It robs a Random Amount from a minAmount to the max Money The Player Have, It Detects if you or The Player Have The Min Amount and If you're Robbing to Yourself, Also it Have A Very Simple         Cooldown System. ex: !rob @Gas_Infinity
```
The backtick character (`) has special meaning in Markdown and denotes inline code. This is (probably) not what you want here.

Reply:
Changed